internal/runners/scripts/edit_test.go

```diff
@@ -100,92 +97,6 @@ func (suite *EditTestSuite) TestCreateScriptFile_Expand() {
 	suite.Equal(v, string(content))
 }
 
-func (suite *EditTestSuite) TestGetOpenCmd_EditorSet() {
```

Comment:
Did you consider moving this test to your new package? If not, is there a way we could unit- or integration-test your new package?